backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala

```diff
@@ -231,14 +228,16 @@ trait StandardAsyncExecutionActor extends AsyncBackendJobExecutionActor with Sta
         |mv $rcTmpPath $rcPath
         |""".stripMargin
       .replace("SCRIPT_PREAMBLE", scriptPreamble)
-      .replace("INSTANTIATED_COMMAND", instantiatedCommand)
+      .replace("INSTANTIATED_COMMAND", instantiatedCommand.commandString)
       .replace("SCRIPT_EPILOGUE", scriptEpilogue))
   }
 
   /** The instantiated command. */
-  lazy val instantiatedCommand: String =
+  lazy val instantiatedCommand: InstantiatedCommand = {
+    val runtimeEnvironment = RuntimeEnvironmentBuilder(jobDescriptor.runtimeAttributes, jobPaths)(standardParams.minimumRuntimeSettings)
     Command.instantiate(
-      jobDescriptor, backendEngineFunctions, commandLinePreProcessor, commandLineValueMapper, RuntimeEnvironmentBuilder(jobDescriptor.runtimeAttributes, jobPaths)(standardParams.minimumRuntimeSettings)).get
+      jobDescriptor, backendEngineFunctions, commandLinePreProcessor, commandLineValueMapper, runtimeEnvironment).toTry.get
```

Comment:
`toTry.get`?

Reply:
This went from `.get` to `.toTry.get` since the callee returns a `Try` rather than an `ErrorOr`. Yes there's a lot of tech debt here, I'm just hoping I didn't make it worse.